### DISCUSSION TITLE
Partially misleading information

### DISCUSSION THREAD

**Discussion by coxantohi** [dotnet/dev-proxy#1228] [Q&A] [answered]
### Type of issue

Missing information

### Feedback

The `Dev Tools don't open in Microsoft Edge on Windows` section's wording leads to believe that this issue would be specific to *Edge*.
I find it worthwhile to mention that irrespective of the `preferredBrowser` setting, all browser instances should be closed before starting Dev Proxy (https://stackoverflow.com/a/72786551/1546142).

### Page URL

https://learn.microsoft.com/en-us/microsoft-cloud/dev/dev-proxy/technical-reference/devtoolsplugin

### Content source URL

https://github.com/MicrosoftDocs/microsoft-cloud/blob/main/docs/dev/dev-proxy/technical-reference/devtoolsplugin.md

### Author

@waldekmastykarz

### Document Id

3f7df5f8-2b7a-4864-46f9-4f02077316a3

**Comment by waldekmastykarz** [comment, marked as answer]
Thanks for your comment. The issue you brought up is not related to having a browser window open. While it's true that you need to close all browser windows, specifically in Edge, even if you have all windows closed, if the startup boost feature is enabled, dev tools still won't work as expected.

**Comment by waldekmastykarz** [comment]
Closing due to lack of response
